src/vm/reflectioninvocation.cpp

```diff
@@ -1714,6 +1723,11 @@ FCIMPL5(void, RuntimeFieldHandle::SetValueDirect, ReflectFieldObject *pFieldUNSA
         if (IsFdLiteral(attr))
```

Comment:
You can delete this check while you are on it. The type loader will not create field descs for `IsFdLiteral` fields.